ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -60,10 +60,9 @@ final class Runner[T <: ReadWriteService, Extra](
         IndexMetadata.read(jdbcUrl).onComplete {
           case Failure(exception) =>
             logger.error("Error while retrieving the index metadata", exception)
-          case Success(None) =>
-            logger.warn("The ledger is not initialized, no metadata to read")
-          case Success(Some(metadata)) =>
+          case Success(metadata) =>
             logger.warn(s"ledger_id: ${metadata.ledgerId}")
+            logger.warn(s"participant_id: ${metadata.participantId}")
```

Comment:
I would prefer avoiding, as much as possible, user confusion due to badly configured logging. It can also be used to ensure that the output stands out. Since the only thing this run mode does is print those four lines and exit, I don't see a specific issue in leaving these at `warn` level.

Reply:
No real issue indeed, it only creates a moment of confusion for users that see the log for the first time and read WARN. It's rather more of a matter of consistency, i.e. "info" should be the threshold used for information (also, log configuration can be so broken, it doesn't even print warnings).